backend/database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -17,9 +17,10 @@ public function up()
             $table->id();
             $table->string('name');
             $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
-            $table->rememberToken();
+            $table->string('avatar')->default('default_profile_avatar.jpg');
```

Comment:
@rogeliojohnoliverio please retain the changes of this file and make another migration if you have changes for users. This is bad practice to touch a migration file that has been already merged. Sample scenario: it might cause trouble in production if some changes are not properly overwritten when we run `php artisan migrate`.